"Speed Up Container Image Builds in Tekton Pipelines" blog post

ggalloro opened the pull request:
This is to add "Speed Up Container Image Builds in Tekton Pipelines" blog post.

Cross post from the article in CDF Blog: https://cd.foundation/blog/2023/10/12/speed-up-container-image-builds-tekton-pipelines/

afrittoli reviewed (approved):
Thanks @ggalloro, this looks good to me!
I appreciate you taking the time to post on the Tekton blog.

I wonder whether it would make sense to add a note at the end like "originally posted @ <link>"

afrittoli commented:
Looking for a second review @AlanGreene @vdemeester

AlanGreene commented:
Thanks @ggalloro. I agree with Andrea's comment, this should include a link to the original post.

Also looks like this relies on content from https://github.com/tektoncd/pipeline/pull/7312 being available. Since it points at the `main` branch rather than a particular commit / tag there's a risk that we'll end up with dead links in the blog post in future.

Would it make sense to include those files in the website repo alongside the blog post instead?

/hold

ggalloro commented:
Thanks @AlanGreene will try to do that.

Any advice to which path to put the sample code in ?

AlanGreene commented:
A sub-folder of the blog post location would make sense to keep it all together

ggalloro commented:
Added a commit to this PR w example code under the blog folder.

Changed the links to example code in the post  with absolute links as https://github.com/tektoncd/website/tree/main/content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/tekton-speed-builds/

Added note with link to original post in CDF blog

ggalloro commented:
Hi @AlanGreene @afrittoli is there any further action needed from my side to get this PR reviewed ?
Many thanks !

AlanGreene commented on content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/index.md:
```diff
@@ -0,0 +1,433 @@
+---
+title: Speeding Up Container Image Builds in Tekton Pipelines
+linkTitle: Speeding Up Container Image Builds in Tekton Pipelines
+date: 2023-11-2
```
The date isn't showing up properly in the preview, I think it requires zero-padding the date

![image](https://github.com/tektoncd/website/assets/2829095/e9705b3f-2f6d-447d-84ea-2d8ec9dd3c1b)
![image](https://github.com/tektoncd/website/assets/2829095/9b1ce9c2-7e90-4c1f-b579-3ac83c913598)

```suggestion
date: 2023-11-02
```

AlanGreene reviewed (changes requested):
Apologies for the delay @ggalloro, I missed the notification.

Content looks good, just a small issue with the date and hopefully then it's good to go 🤞

AlanGreene reviewed (approved):
Thanks @ggalloro 
/lgtm

AlanGreene commented:
/hold cancel

afrittoli commented:
🎉
